ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -203,131 +207,119 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
       },
       {
         // Logout/Exit
-        value: I18n.t("profile.main.logout"),
-        description: I18n.t("profile.logout.menulabel"),
-        onPress: this.onLogoutPress,
-        hideChevron: true
+        value: I18n.t("profile.logout.menulabel"),
+        hideChevron: true,
+        iconColor: "error-850",
+        testID: "logoutButton",
+        icon: "logout",
+        renderValue: v => (
+          <>
+            <VSpacer />
+            <H6 color="error-850">{v}</H6>
+            <VSpacer />
+          </>
+        ),
+        onPress: onLogoutPress
```

Comment:
I know that in the Figma project there's a custom version of the `ListItemInfo` component, but please use `ListItemAction` instead of the custom `ListItemNav` here. It has the proper color values and it's also dark mode compliant (for future developments). Check out the **List Items** page in the DS section for reference.

Reply:
Thank you for your feedback! I will replace it with `ListItemAction` as soon as possible!